### PR TITLE
feat: volunteer applications

### PR DESCRIPTION
**Pull request by jadeallencook**
Updated the become a volunteer form:

Screenshot 2023-02-26 at 10 25 52 PM

You can see the applications on the dashboard now too:

Screenshot 2023-02-26 at 10 25 25 PM

Clicking on an application opens it in a modal:

Screenshot 2023-02-26 at 10 25 37 PM

There's also some reformatting and spelling updates too.

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-npx pretty-quick --staged
+# npx pretty-quick --staged
```
Comment (asbhaskar) [inline .husky/pre-commit:4]: Extra comment here

File: src/components/ApplicationModal/style.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const style = () => ({
+    '.modal-title': {
+        color: 'var(--primary)',
+        fontWeight: '900',
+        textTransform: 'capitalize',
+    },
+});
+
+export default style;
```
Comment (asbhaskar) [inline src/components/ApplicationModal/style.ts:9]: `src/RequestModal` uses these same styles but they're inline. Should we create a class for this in `theme.css` under `.modal`?

**Review by asbhaskar — COMMENTED** [review]
Looks great!

**Comment by claytonleonardcook** [comment]
LGTM

**Review by claytonleonardcook — APPROVED** [review]
LGTM

**Pull request merged**